src/java.base/share/classes/java/io/File.java

```diff
@@ -42,7 +42,9 @@
 import java.net.URISyntaxException;
 import java.nio.file.FileStore;
 import java.nio.file.FileSystems;
+import java.nio.file.LinkOption;
```

Comment:
Why are the additional imports needed?

Reply:
Ah, I think ide must do this automatically... I never did that myself. I see a few places in the comment. I will remove them.